src/entity/ReadingStreakActions.ts

```diff
@@ -0,0 +1,24 @@
+import { Column, Entity, OneToOne, PrimaryColumn } from 'typeorm';
+import { UserStreak } from './user';
+
+export enum ReadingStreakActionType {
+  Recover = 'recovered',
+}
+
+@Entity()
+export class ReadingStreakActions {
+  @PrimaryColumn({ length: 36, default: '' })
+  id: string;
+
+  @OneToOne(() => UserStreak, (userStreak) => userStreak.userId, {
+    lazy: true,
+    onDelete: 'CASCADE',
+  })
+  userStreak: Promise<UserStreak>;
+
+  @Column({ type: 'timestamptz', default: () => 'now()' })
+  timestamp: Date;
+
+  @Column({ type: 'enum', enum: ReadingStreakActionType })
```

Comment:
I think we prefer storing as strings but forcing enum on graph side (this makes it a bit more flexible to add/change stuff later on)